backend/src/routes.rs

```diff
@@ -0,0 +1,154 @@
+use actix_web::{
+    delete, get, post, put,
+    web::{self},
+    HttpResponse, Scope,
+};
+use subxt::ext::sp_core::H256;
+use uuid::Uuid;
+
+use crate::{
+    database::{
+        dto::{AttestationRequest, Credential, Query, UpdateAttestation},
+        querys::{
+            approve_attestation_request, can_approve_attestation, can_revoke_attestation,
+            delete_attestation_request, get_attestation_request_by_id, get_attestation_requests,
+            get_attestations_count, insert_attestation_request, revoke_attestation_request,
+            update_attestation_request,
+        },
+    },
+    error::AppError,
+    AppState,
+};
+
+#[get("/{attestation_request_id}")]
+async fn get_attestation(
+    path: web::Path<Uuid>,
+    state: web::Data<AppState>,
+) -> Result<HttpResponse, AppError> {
+    let attestation_id = path.into_inner();
+
+    let attesttation = get_attestation_request_by_id(&attestation_id, &state.db_executor).await?;
+
+    Ok(HttpResponse::Ok().json(serde_json::json!(attesttation)))
```

Comment:
Why use the `serde_json::json!` here? That seems like an overkill for simply transforming it to a json::Value?

Reply:
I updated it to the `to_string` function: [35574e0](https://github.com/KILTprotocol/attester/pull/1/commits/35574e09be60b7a3fae48e44320e73c232051780).

What is the problem with the json! macro?

Reply:
What is the problem with `serde_json::json!`? 

Transformed the structs to strings now: [35574e0](https://github.com/KILTprotocol/attester/pull/1/commits/35574e09be60b7a3fae48e44320e73c232051780)

Reply:
AFAIK you can simply do:

```
    Ok(HttpResponse::Ok().json(attesttation))
```

The problem with `json!` is, that this is used to build json documents in rust. [Like that:](https://docs.rs/serde_json/latest/serde_json/#constructing-json-values)

```
json!({
        "name": "John Doe",
        "age": 43,
        "phones": [
            "+44 1234567",
            "+44 2345678"
        ]
    });
```

So it's purpose is to rather build a bigger structure, that to transform a single value.